bin/deploy-vhost

```diff
@@ -636,6 +647,17 @@ END
     }
 }
 
+sub get_conf_ugly_file_from_servers_dir {
+  my ($conf_ugly_part) = @_;
+  my $conf_ugly_file = "$servers_dir/vhosts/$vcspath/$vhost/$conf_ugly_part.ugly";
+  $conf_ugly_file = "$servers_dir/vhosts/$vcspath/$conf_ugly_part.ugly" unless -e $conf_ugly_file;
+  $conf_ugly_file = "$servers_dir/vhosts/${vcspath}_$conf_ugly_part.ugly" unless -e $conf_ugly_file;
+  # update the conf/general (or whatever) template in private git repo
+  chdir_verbose(__LINE__, "$servers_dir/vhosts");
+  shell("git", "pull", "-q", "origin");
```

Comment:
It seems odd to do the git pull after the checking the files exist in that location. I realise this is what it was doing previously, but I'm not sure it's right :)